src/picongpu/include/particles/ParticlesFunctors.hpp

```diff
@@ -155,29 +155,66 @@ struct CallUpdate
     }
 };
 
-/* Tests if species can be ionized and calls the function to do that */
+/** \struct CallIonization
```

Comment:
just use the doxygen syntax we usually use:
```c++
Headline

longer description
```

Writing the name of the class as the headline is not ideal, it does not add documentation value